composer.json

```diff
@@ -57,6 +57,9 @@
 			"ignore-as-root": false,
 			"web-dir": ".Build/Web",
 			"app-dir": ".Build"
+		},
+		"branch-alias": {
+			"dev-main": "4.0.x-dev"
```

Comment:
Would suggest to go with `4.x.x-dev` instead. More open and do not limit yet on minor version.

But also fine with it.